olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/model/Messages.h

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <olp/dataservice/read/DataServiceReadApi.h>
+#include <olp/dataservice/read/model/Data.h>
+#include <olp/dataservice/read/model/StreamOffsets.h>
+#include <boost/optional.hpp>
+
+namespace olp {
+namespace dataservice {
+namespace read {
+namespace model {
+
+/**
+ * @brief Class, which encapsulates info about actual data content.
+ */
+class DATASERVICE_READ_API Metadata final {
+ public:
+  /**
+   * @return Returns the
```

Comment:
Please also provide a ```@brief``` field.
Same for all other getters below.

Reply:
Oops, looks like I added `@brief ` to all setters and getters. Anyway, please, double check.